NG new Typings Error

> Please provide us with the following information:
> ---------------------------------------------------------------
1. OS?  Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)
Ubuntu 16.04

2. Versions. Please run `ng --version`. If there's nothing outputted, please
   run in a Terminal: 
        node --version
   And paste the result here.
angular-cli: 1.0.0-beta.6
node: 6.2.1
os: linux x64

3. Repro steps. Was this an app that wasn't created using the CLI? What change did you
   do on your code? etc.
ng new TestApp

4. The log given by the failure. Normally this include a stack trace and some
   more information.

```
ricardo@Ricardo:~/WebstormProjects/temp$ ng new MyApp
Could not start watchman; falling back to NodeWatcher for file system events.
Visit http://ember-cli.com/user-guide/#watchman for more info.
installing ng2
  create .editorconfig
  create README.md
  create src/app/app.component.css
  create src/app/app.component.html
  create src/app/app.component.spec.ts
  create src/app/app.component.ts
  create src/app/environment.ts
  create src/app/index.ts
  create src/app/shared/index.ts
  create src/favicon.ico
  create src/index.html
  create src/main.ts
  create src/system-config.ts
  create src/tsconfig.json
  create src/typings.d.ts
  create angular-cli-build.js
  create angular-cli.json
  create config/environment.dev.ts
  create config/environment.js
  create config/environment.prod.ts
  create config/karma-test-shim.js
  create config/karma.conf.js
  create config/protractor.conf.js
  create e2e/app.e2e-spec.ts
  create e2e/app.po.ts
  create e2e/tsconfig.json
  create e2e/typings.d.ts
  create .gitignore
  create package.json
  create public/.npmignore
  create tslint.json
  create typings.json
Successfully initialized git.
⠼ Installing packages for tooling via npmsh: 1: typings: not found
File: sh
@angular/router@3.0.0-alpha.3 postinstall: `typings install`
spawn ENOENT
Error: @angular/router@3.0.0-alpha.3 postinstall: `typings install`
spawn ENOENT
    at ChildProcess.<anonymous> (/home/ricardo/.npm-global/lib/node_modules/angular-cli/node_modules/npm/lib/utils/spawn.js:17:16)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:852:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)

```

5. Mention any other details that might be useful.
> ---------------------------------------------------------------
> > Thanks! We'll be in touch soon.

Windows 7 gives me a slightly different error, but the main cause is the same.  This is a clean install of Node 6.2.1 with npm 3.9.3. Installed Node, then npm install -g angular-cli

Command
ng new TestApp
...
  create tslint.json
  create typings.json
Successfully initialized git.
- Installing packages for tooling via npm'typings' is not recognized as an internal or external command,
operable program or batch file.
@angular/router@3.0.0-alpha.3 postinstall: `typings install`
Exit status 1
Error: @angular/router@3.0.0-alpha.3 postinstall: `typings install`
Exit status 1
    at EventEmitter.<anonymous> (C:\Users\stevens\AppData\Roaming\npm\node_modul
es\angular-cli\node_modules\npm\lib\utils\lifecycle.js:217:16)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:191:7)
    at ChildProcess.<anonymous> (C:\Users\stevens\AppData\Roaming\npm\node_modul
es\angular-cli\node_modules\npm\lib\utils\spawn.js:24:14)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:852:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)

Running  ng --version gives error:
module.js:442
    throw err;
    ^

Error: Cannot find module 'ember-cli/lib/utilities/find-build-file'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\stevens\Development\AgentUI\node_modules\ang
ular-cli\lib\cli\index.js:48:31)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)

All ng (ng --version, ng serve, ng test) return the same error.  Everything was installed under npm in the Windows Command window running as an Administrator with the shortcut being executed as Run as Adminstrator from the right click menu.

--- a/addon/ng2/blueprints/ng2/files/package.json
+++ b/addon/ng2/blueprints/ng2/files/package.json
@@ -19,7 +19,7 @@
     "@angular/http": "2.0.0-rc.2",
     "@angular/platform-browser": "2.0.0-rc.2",
     "@angular/platform-browser-dynamic": "2.0.0-rc.2",
-    "@angular/router": "3.0.0-alpha.5",
+    "@angular/router": "3.0.0-alpha.7",
     "es6-shim": "0.35.1",
     "reflect-metadata": "0.1.3",
     "rxjs": "5.0.0-beta.6",

